astropy/modeling/core.py

```diff
@@ -1074,22 +1074,28 @@ def evaluate(self, *args, **kwargs):
 
     def render(self, out=None, coords=None):
         """
-        Evaluates a model on an input array. Evaluation is limited to
-        a bounding box if the `Model.bounding_box` attribute is set.
+        Evaluates a model on an input array.  I.e., "discretizes" the model.
```

Comment:
Good point, @astrofrog.  This just came up in #4572, actually.  Will clear that up.

I wonder if we should consider adding a switch in `render` to allow it to do `discretize_model`, too?  I'm thinking the typical user (me included, in this case!) won't think to look at convolution for this.